src/components/Toast/Toast.tsx

```diff
@@ -16,28 +15,16 @@ export type TransactionToastComponentProps = ToastComponentProps & {
 
 export const Toast: FC<PropsWithChildren<ToastComponentProps>> = ({
   className,
-  isVisible,
   children,
 }) => {
   return (
-    <Transition
```

Comment:
Did you mean to remove the transition?

Reply:
Yep, I used the hot-react-toast default transition. Actually, I first modified the transition to hasten the close button action, I now believe the default behavior to be acceptable.